torch/_prims_common/__init__.py

```diff
@@ -1474,8 +1474,18 @@ def elementwise_dtypes(
     # See https://dev-discuss.pytorch.org/t/delving-into-what-happens-when-you-import-torch/1589
     import sympy
 
+    from torch._inductor.codegen.common import CSEVariable
+
```

Comment:
We should not leak concepts from torchinductor into prims dtype promotion. You can avoid this by doing what inductor does [here]( https://github.com/pytorch/pytorch/blob/c272526ea54b8063c596addb48cd3a87ab91a5ef/torch/_inductor/lowering.py#L224).